Objects/typeobject.c

```diff
@@ -5625,6 +5625,12 @@ object_richcompare(PyObject *self, PyObject *other, int op)
     return res;
 }
 
+PyObject*
+_Py_object_tp_richcompare(PyObject* self, PyObject* other, int op)
```

Comment:
Can you rename the function to `_Py_object_tp_richcompare`?

Reply:
It's already named `_Py_object_tp_richcompare`. Maybe you misunderstood something?

Reply:
Sorry, I mean to rename it to `_Py_object_richcompare()`: remove `tp_`.

Maybe to avoid confusion with PyObject_RichCompare(), maybe it should be renamed to: `_Py_BaseObject_RichCompare()`.